src/reducers/settings.js

```diff
@@ -346,6 +348,13 @@ const handlers: Object = {
     ...state,
     lastSeenDevice: dmi,
   }),
+  SET_LAST_CONNECTED_DEVICE: (
```

Comment:
Seems like a lost chance to use last seen device and enhance it, no?

Reply:
sure but the scope of that change would require to update all uses of last seen device to use Device instead of DeviceModelInfo
which wasnt as easy to do as far as i remember :thinking: